### ISSUE TITLE
Broken after lastest update

### ISSUE THREAD

**Issue by davidjrea** [nicholasehamilton/ggtern#37]
I updated all of my packages (and R to 3.5.1) which seems to have broken ggtern. I get the following error with both the CRAN and development version when trying to run the examples.

Error in f(..., self = self) : unused argument (plot$coordinates)

**Comment by jamesdalg** [comment]
Yes, I get the exact same issue and I even downgrading to ggplot 2.2.1 doesn't fix it.
